tools/project_manager/editproject.php

```diff
@@ -169,6 +169,7 @@ function set_from_nothing()
         $this->comments         = '';
         $this->clearance        = '';
         $this->postednum        = '';
+        $this->charsuites       = ['basic-latin'];
```

Comment:
While it will for pgdp.net, we can't guarantee that 'basic-latin' is always going to be enabled for all sites and this would cause problems. You could either do the "if basic-latin is an enabled charsuite then use it" logic here and on line 210 or in a helper function in this file that does the same thing.

Reply:
ok, went ahead and use empty if basic-latin isn't enabled. Then they'll get an error if they save without one.